vendors/microchip/secure_elements/app/example_trust_chain_tool/README.md

```diff
@@ -25,47 +25,75 @@ PKI system) and an intermediate certificate authority that mirrors the signing
 hardware security modules (HSM) that are used in the Microchip facility during
 manufacturing of security devices.
 
-### Create the Root CA
+In this flow, you will create a Root CA, a Signing CA, and finally a device certificate
+in what is known as a "chain of trust".
+
+Your Root CA will be self-signed at the top of your chain of trust.  
+
+Your Signing Certificate will be trusted by your Root CA, and will be used to issue your device certificate.
+
+Your Device Certificate will be unique per device, and is trusted by your Signing Certificate.
 
+### Create the Root CA
+First, create your Root CA.  
 ```
-> ca_create_root.py 
+>python ca_create_root.py 
 ```
 
 ### Create the Signing CA
+Next, create your Signing CA.  This CA will be registered with your AWS IoT account.  
+In order to prove your ownership of the Signing CA, obtain a registration code from AWS.
 
 ```
-> ca_create_signer.py <code_from_aws>
+aws iot get-registration-code
```

Comment:
We should provide one or more links to external documentation for setting up the CLI and for understanding this particular API. Otherwise, this seems to come out of nowhere.

Reply:
yes done. and also a mention that openssl should be installed.